features/step_definitions/auction_create_steps.rb

```diff
@@ -11,7 +11,7 @@
 end
 
 Given(/^there is a closed auction$/) do
-  @auction = FactoryGirl.create(:auction, :closed, :with_bids)
+  @auction = FactoryGirl.create(:auction, :closed, :with_bids, :c2_budget_approved)
```

Comment:
what about created a new step called "there is a closed default purchase card auction"?

We've been using this step as-is for other tests and the c2 status was not necessary. If we need to c2 cudget approved status does matter for one test, separate step might be nice